site/docs/id/hosting/fly.md

```diff
@@ -0,0 +1,436 @@
+2. Buat sebuah file dengan nama `app.ts` atau `app.js`, ataupun nama lainnya sesuai dengan keinginanmu (tetapi kamu harus mengingatnya karena nanti file tersebut akan digunakan sebagai file deploy utama). File tersebut berisikan:
+
+<CodeGroup>
+<CodeGroupItem title="Deno" active>
+
+```ts{11}
+import { serve } from "https://deno.land/std/http/server.ts";
+import { webhookCallback } from "https://deno.land/x/grammy/mod.ts";
+// Kamu mungkin perlu mengubah ini agar object bot-mu bisa di-import.
+import { bot } from "./bot.ts";
+
+const port = 8000;
+const handleUpdate = webhookCallback(bot, "std/http");
+
+serve(async (req) => {
+  const url = new URL(req.url);
+  if (req.method === "POST" && url.pathname.slice(1) === bot.token) {
+    try {
+      return await handleUpdate(req);
+    } catch (err) {
+      console.error(err);
+    }
+  }
+  return new Response();
+}, { port });
+```
+
+</CodeGroupItem>
+<CodeGroupItem title="Node.js" active>
+
+```ts{10}
+import express from "express";
+import { webhookCallback } from "grammy";
+// Kamu mungkin perlu mengubah ini agar object bot-mu bisa di-import.
+import { bot } from "./bot";
+
+const port = 8000;
+const app = express();
+
+app.use(express.json());
+app.use(`/${bot.token}`, webhookCallback(bot, "express"));
+app.use((_req, res) => res.status(200).send());
+
+app.listen(port, () => console.log(`listening on port ${port}`));
+```
+
+</CodeGroupItem>
+</CodeGroup>
+
+Kami menganjurkan kamu untuk menaruh handler di direktori rahasia alih-alih menempatkanya di root (`/`).
+Di contoh kali ini, kita menggunakan token bot (`/<token bot>`) sebagai direktori rahasianya (perhatikan baris kode yang disorot).
+
+### Long Polling
+
+Buat sebuah file dengan nama `app.ts` atau `app.js`, ataupun nama lainnya sesuai dengan keinginanmu (tetapi kamu harus mengingatnya karena nanti file tersebut akan digunakan sebagai file deploy utama).
+File tersebut berisikan:
+
+<CodeGroup>
+<CodeGroupItem title="Deno" active>
+
+```ts
+import { Bot } from "https://deno.land/x/grammy/mod.ts";
+
+// Di sini, kita mengambil token bot dari variable "BOT_TOKEN".
+const bot = new Bot(Deno.env.get("BOT_TOKEN"));
+
+bot.command(
+  "start",
+  (ctx) => ctx.reply("Aku berjalan di Fly menggunakan long polling!"),
+);
+
+Deno.addSignalListener("SIGINT", () => bot.stop());
+Deno.addSignalListener("SIGTERM", () => bot.stop());
+
+bot.start();
+```
+
+</CodeGroupItem>
+<CodeGroupItem title="Node.js">
+
+```ts
+import { Bot } from "grammy";
+
+// Di sini, kita mengambil token bot dari variable "BOT_TOKEN".
+const bot = new Bot(process.env.BOT_TOKEN);
+
+bot.command(
+  "start",
+  (ctx) => ctx.reply("Aku berjalan di Fly menggunakan long polling!"),
+);
+
+process.once("SIGINT", () => bot.stop());
+process.once("SIGTERM", () => bot.stop());
+
+bot.start();
+```
+
+</CodeGroupItem>
+</CodeGroup>
+
+Seperti yang terlihat di baris kode yang disorot di atas, kita telah mengambil informasi sensitif (token bot kamu) dari environment variables.
```

Comment:
```
Seperti yang terlihat di baris kode yang disorot di atas
```

That's sentence looks like something bit odd maybe because `yang` and another `yang` too close.

How about this sugestion?

```suggestion
Lihat baris kode yang disorot di atas, kita telah mengambil informasi sensitif (token bot kamu) dari environment variables.
```

Reply:
Hmm.. you're right. That was not efficient. Perhaps we should use `Perhatikan` instead of `Lihat`? The word `Lihat` sounds little aggresive to me :sweat_smile: .

Reply:
Ok.